### ISSUE TITLE
RS90 screen will not shutdown properly

### ISSUE THREAD

**Issue by voodoo47** [OpenDingux/buildroot#137]
After powering down using the power off icon, the screen clearly stays lit until the actual ON/OFF switch on the side is flipped.

https://www.youtube.com/watch?v=vBl2renriaY

**Comment by pcercuei** [comment]
Can you try with the latest build? Does it still happen?

**Comment by voodoo47** [comment]
Yes, despite the green led turning off once the operating system shuts down, the screen very clearly stays on until the hardware switch is flipped to the OFF position.

**Comment by SnDream** [comment]
Maybe it's related to the hardware version, I have multiple r90's with the same software version, some have the LCD turn off after power off, some don't.
